edx-platform/pearson-vue-theme/lms/templates/dashboard.html

```diff
@@ -113,18 +113,54 @@
 
 <div class="dashboard-notifications" tabindex="-1">
     <div class="page-banner">
-        <div class="user-messages" role="complementary" aria-label="messages">
-            <ul class="user-messages-ul">
-                    <li>
-                        <div class="alert alert-warning" role="alert">
-                            <span class="icon icon-alert fa fa fa-warning" aria-hidden="true"></span>
-                            <p class="lh-lg">
-                                CertPREP is getting a new name! In addition to the change you can see in our banner, you'll continue to see ongoing changes in our courseware and communications over the few months as we transition to our new "Pearson Skilling Suite" name.
-                            </p>
-                        </div>
-                    </li>
-            </ul>
-        </div>
+      <div class="user-messages" role="complementary" aria-label="messages">
+        <ul class="user-messages-ul">
+            <li>
+                <div class="alert alert-warning" role="alert">
+                    <span class="icon icon-alert fa fa-warning" aria-hidden="true"></span>
+                    <p class="lh-lg banner-message">
+                      CertPREP is getting a new name! In addition to the change you can see in our banner, you'll continue to see ongoing changes in our courseware and communications over the next few months as we transition to our new "Pearson Skilling Suite" name.
```

Comment:
Why you don't use a variable to configure the text? When we don't need this banner or reuse the component for other cases, is always necessary to modify the template and is not the best way

Reply:
Where could I configure that variable? I tried to use a custom variable inside of the site config ( outside of the mfe_config block ) but it didn't work

Reply:
You have to use configuration_helpers, this is an example how to use it: https://github.com/Pearson-Advance/openedx-themes/blob/ee54283898324447887fac9f7c4bb51f120d03b3/edx-platform/pearson-vue-theme/lms/templates/dashboard.html#L172  
the second argument is the default value, you can define it as settings.MAINTENANCE_BANNER_TEXT